cli/src/commands/deployments/register.rs

```diff
@@ -252,7 +252,7 @@ pub async fn run_register(State(env): State<CliEnv>, discover_opts: &Register) -
     if !updated.is_empty() {
         c_println!();
         // used to resolving old endpoints.
```

Comment:
Yes, but i couldn't replace all automatically, I had to check every change myself because we often used the term `endpoint` for `service endpoint`, but endpoint is a term meaningful on its own in other contexts.